### ISSUE TITLE
You look like you are going through some tough stuff

### ISSUE THREAD

**Issue by nathanpeck** [botupdate/botupdate#6]
Hi,

I saw your graffiti in my neighborhood with a link to this Github. You look like you are going through some tough stuff. There are some things that you can do on your own to help yourself feel better:

- Eat healthy food
- Get some exercise
- Reach out to friends and family if you can
- Avoid alcohol, marijuana, or other hard drugs

But I feel like you are trying to reach out to people with your graffiti because you need more help than what you can do on your own. I just want you to know that people in your community see you need some help. You can talk to professionals about what you are going through: a doctor, therapist, or crisis counselor.

If you ever need some help please consider using one of the following free resources:

- Crisis Text Line: Text CONNECT to 741741 or visit: https://www.crisistextline.org/textline/
- National Suicide Prevention Lifeline: 1-800-273-8255

There are a lot of telephone hotlines with people who can answer right away to help you get the resources you need, or just listen if you need to talk.

**Comment by iamme-andU** [comment]
I’m telling you right now, this man knows what he is talking about! I just come across this an hour ago. I too have this same problem!  I’m so amazed right now anyone else is having the issue too! 

I even emailed him! I recently just sold about $3,500 dollars To an OTA machine for a lousy $400 in iPhones, iPads, Samsung’s LG’s and I’m soon going to be trading my car into also because they have hacked my app that I used for my cars onstar on my phone for my car diagnostics and door locks and Sirius xm.
Now I’m forced to listened to them and the songs they play while making fun of me and my life.
It has made my self confidence drop times 10.
I traded 4 brand new IPhones 11’s back to Verizon within 4 days of purchasing them because they were hacked the minute I took them out of the box!
They claim on my end they are a relative of Adolf Hitler  and basically they think they are going to take over the world! They won’t stop until they are dead or we are! I started getting stalked at work after taking numerous of photos of the stuff happening on my things. Eventually a man showed up there telling me if I didn’t go travel he was going to hurt my family! I believe whole heartily this Does have something to do with trafficking! 
They print my personal information out in webpages. I’m praying now someone can help me!

I purchased geek squad and day after day, I lived there for weeks trying to get to the bottom of it. They eventually handed me a name of an FBI agent and a address to the FBI building! They eventually Told me to go straight to them with all my technology, because they feared for my life! They took one look at that analytics and knew it wasn’t right.
 I made 3 price reports any way more than to apple and numerous of different cell phone company’s and home phone company’s.
This is the worst thing I have battled in my life! 
I’m am very very thankful to have come across this! So before anyone degrades this man agin for this you might want to consider this....... Because of this past hour of reading this story I’m so thankful and he just saved a life.... Sooo Until you live the shoes. Don’t you dare comment with bashing. People who have nothing more than to argue someone about something they obviously have no idea about, should never!!! That does nothing for one confidence I may add!!!  But now I just want to thank this man he has just lifted 100lbs of my shoulders! Thank you Thank you! 
I apologize ahead of time if I have typos!
